corehq/apps/userreports/exceptions.py

```diff
@@ -82,12 +82,11 @@ class InvalidDataSourceType(UserQueryError):
 
 
 class ValidationError(UserReportsError):
-    def __init__(self, name, message):
-        self.name = name
-        self.message = message
+    def __init__(self, errors):
+        self.errors = errors
 
     def __str__(self):
-        return self.message
+        return '\n'.join("{}: {}".format(e[0], e[1]) for e in self.errors)
```

Comment:
there may be some value in using a named tuple later. the surface area for this is small enough that I don't think its wortwhile at the moment